src/components/Landing/components/ContactUs/index.scss

```diff
@@ -0,0 +1,76 @@
+@import "../../../../constants/constants.scss";
+
+.contactus-wrapper {
+  padding: 50px 0;
+  background: linear-gradient(90.04deg, #160055 -5.45%, rgba(22, 23, 43, 0) 99.96%, rgba(22, 23, 43, 0) 99.96%);
+  .section-content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .contactus-form {
+      width: 100%;
+      max-width: 386px;
+      &__title {
```

Comment:
I would strongly recommend to use full `snake case` for example instead of `&__title` so `contactus-form__title`. This way it's super easy to find the right class in case of debugging/making changes. Besides I think it's very confusing that half of the project is written in certain way and the other in different way - so in my opinion we have to go with on way.